webapp/src/containers/modals/LinkModal/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {useEffect} from 'react';
+import {useDispatch} from 'react-redux';
+
+import Modal from 'components/modal';
+import Form from 'components/form';
+
+import plugin_constants from 'plugin_constants';
+
+import {toggleShowLinkModal} from 'reducers/linkModal';
+import {getLinkModalState} from 'selectors';
+
+import usePluginApi from 'hooks/usePluginApi';
+import useForm from 'hooks/useForm';
+import useApiRequestCompletionState from 'hooks/useApiRequestCompletionState';
+
+const LinkModal = () => {
+    const {linkProjectModal} = plugin_constants.form;
+
+    // Hooks
```

Comment:
```suggestion
```

Reply:
we are writing types for each block as hooks, state variables, etc
why should we remove it?

Reply:
It just doesn't seem useful. You can keep it if you want.